### ISSUE TITLE
XpathHandler QueryAll Syntax error: Unexpected token '}'

### ISSUE THREAD

**Issue by nunutu29** [hardkoded/puppeteer-sharp#2251]
### Description

Using the function QuerySelectorAllAsync with the following selector:
xpath///one-app-nav-bar-item-root/button[ count(.//*[contains(@icon-name, 'close')]) > 0 ]

I am getting the exception: Evaluation failed: SyntaxError: Unexpected token '}'
It looks like there are too many parenthesis at the end of the QueryAll Handler.

https://github.com/hardkoded/puppeteer-sharp/blob/8603d0e12a4370844ea669759744bebd8fa09b77/lib/PuppeteerSharp/CustomQueriesManager.cs#L219C16-L219C16

### Expected Behaviour:
And empty array.

### Actual Behavior:
```
PuppeteerSharp.EvaluationFailedException
  HResult=0x80131500
  Message=Evaluation failed: SyntaxError: Unexpected token '}'
  Source=PuppeteerSharp
  StackTrace:
   at PuppeteerSharp.ExecutionContext.<ExecuteEvaluationAsync>d__35.MoveNext() in /home/runner/work/puppeteer-sharp/puppeteer-sharp/lib/PuppeteerSharp/ExecutionContext.cs:line 206
   at PuppeteerSharp.ExecutionContext.<EvaluateFunctionInternalAsync>d__34.MoveNext() in /home/runner/work/puppeteer-sharp/puppeteer-sharp/lib/PuppeteerSharp/ExecutionContext.cs:line 192
   at PuppeteerSharp.ExecutionContext.<EvaluateFunctionHandleAsync>d__23.MoveNext() in /home/runner/work/puppeteer-sharp/puppeteer-sharp/lib/PuppeteerSharp/ExecutionContext.cs:line 61
   at PuppeteerSharp.CustomQueriesManager.<>c__DisplayClass13_0.<<CreatePuppeteerQueryHandler>b__2>d.MoveNext() in /home/runner/work/puppeteer-sharp/puppeteer-sharp/lib/PuppeteerSharp/CustomQueriesManager.cs:line 278
   at PuppeteerSharp.IsolatedWorld.<QuerySelectorAllAsync>d__36.MoveNext() in /home/runner/work/puppeteer-sharp/puppeteer-sharp/lib/PuppeteerSharp/IsolatedWorld.cs:line 212
  ....
```

### Versions

* Which version of PuppeteerSharp are you using? 
v10.1.0
* Which .NET runtime and version are you targeting? E.g. .NET framework 4.6.1 or .NET Core 2.0.
.NET CORE 6.0

**Comment by kblok** [comment]
Thank you for the report!

**Comment by kblok** [comment]
@nunutu29 fixed in 10.1.1.
